components/brave_wallet/browser/brave_wallet_service.cc

```diff
@@ -40,6 +40,7 @@ void BraveWalletService::RegisterProfilePrefs(PrefRegistrySimple* registry) {
   registry->RegisterIntegerPref(kBraveWalletDefaultKeyringAccountNum, 0);
   registry->RegisterBooleanPref(kShowWalletIconOnToolbar, true);
   registry->RegisterBooleanPref(kBraveWalletBackupComplete, false);
+  registry->RegisterStringPref(kBraveWalletLastUnlockYMD, "");
```

Comment:
Any reason why not to use `registry->RegisterTimePref`

Reply:
Updated to do that